rdt/transformers/base.py

```diff
@@ -292,10 +292,10 @@ def _add_columns_to_data(data, transformed_data, transformed_names):
         Returns:
             ``pandas.DataFrame`` with the new columns added.
         """
-        if isinstance(transformed_data, (pd.Series, np.ndarray)):
-            transformed_data = pd.DataFrame(transformed_data, columns=transformed_names)
```

Comment:
In the latest version of pandas, this raises an error if `columns=[]`, while before it used to return an empty dataframe. This causes the `RandomLocationGenerator` to fail in SDV Enterprise, so it needs to be fixed.